apps/ui/src/views/settings/ApiKeys.tsx

```diff
@@ -63,53 +69,68 @@ export default function ProjectApiKeys() {
                 open={Boolean(editing)}
                 onClose={() => setEditing(null)}
             >
-                <FormWrapper<ProjectApiKey>
-                    onSubmit={
-                        async ({ id, name, description, scope }) => {
-                            if (id) {
-                                await api.apiKeys.update(project.id, id, { name, description })
-                            } else {
-                                await api.apiKeys.create(project.id, { name, description, scope })
+                {
+                    editing && (
+                        <FormWrapper<ProjectApiKey>
+                            onSubmit={
+                                async ({ id, name, description, scope }) => {
+                                    if (id) {
+                                        await api.apiKeys.update(project.id, id, { name, description })
+                                    } else {
+                                        await api.apiKeys.create(project.id, { name, description, scope })
+                                    }
+                                    await state.reload()
+                                    setEditing(null)
+                                }
                             }
-                            await state.reload()
-                            setEditing(null)
-                        }
-                    }
-                    defaultValues={editing ?? { scope: 'public' }}
-                    submitLabel={editing ? 'Save' : 'Create'}
-                >
-                    {
-                        form => (
-                            <>
-                                <TextField
-                                    form={form}
-                                    name="name"
-                                    label="Name"
-                                    required
-                                />
-                                <TextField
-                                    form={form}
-                                    name="description"
-                                    label="Description"
-                                    textarea
-                                />
-                                {
-                                    !!(editing && !editing.id) && (
-                                        <OptionField
-                                            form={form}
-                                            name="scope"
-                                            label="Scope"
-                                            options={[
-                                                { key: 'public', label: 'Public' },
-                                                { key: 'secret', label: 'Secret' },
-                                            ]}
-                                        />
+                            defaultValues={editing}
+                            submitLabel="Create"
```

Comment:
Looks like these two lines got lost in the merge. The default values and what the submit button says